src/scheduler/fifo.c

```diff
@@ -603,12 +603,6 @@ intermediate_schedule(int sd, char *jobid)
 	int ret; /* to re schedule or not */
 	int cycle_cnt = 0; /* count of cycles run */
 
-	/*
-	 * This is required since there is a probability that scheduler's configuration has been changed at
-	 * server through qmgr.
-	 */
-	update_svr_schedobj(connector, 0, 0);
```

Comment:
Now that we're not calling this here, what happens if a multi-sched comes up when the server is down.  We no longer call SCH_CONFIGURE_ATTRS on recovery, so how will a scheduler initially get its sched_priv/sched_log directories.

Reply:
As soon as Server comes up the function update_svr_schedobj gets called(from pbs_sched.c) which in turn calls the function sched_settings_frm_svr because of the command SCH_SCHEDULE_FIRST. This is how Multisched gets its log and priv directories configured at Server. Till the time Server comes up it assumes that sched_logs_sc1 and sched_priv_sc1 are its desired directories if Multisched is named as sc1. Hope this answers your question.